Sources/Confidence/ConfidenceClient/RemoteConfidenceClient.swift

```diff
@@ -38,21 +38,27 @@ public class RemoteConfidenceClient: ConfidenceClient {
             sendTime: timeString,
             sdk: Sdk(id: metadata.name, version: metadata.version)
         )
-
         do {
             let result: HttpClientResult<PublishEventResponse> =
             try await self.httpClient.post(path: ":publish", data: request)
             switch result {
             case .success(let successData):
-                guard successData.response.status == .ok else {
+                guard let status = successData.response.status else {
```

Comment:
let's try to replace this line with `successData.response.statusCode` and we can remove the guard and else in this case.